test-framework/junit5/src/main/java/io/quarkus/test/junit/NonQuarkusTestLoggingEnabler.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.test.junit;
+
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import io.quarkus.test.common.PropertyTestUtil;
+
+public class NonQuarkusTestLoggingEnabler implements BeforeAllCallback {
+
+    // warning: this name is also used in junit5-internal
+    //private static final String SYSPROP_DISABLE = NonQuarkusTestLoggingEnabler.class.getName() + ".disable";
+
+    private static boolean initialized;
+
+    //    public static void disable() {
+    //        if (!initialized) {
+    //            initialized = true;
+    //            //System.setProperty(SYSPROP_DISABLE, "true");
+    //        }
+    //    }
+
+    @Override
+    public void beforeAll(ExtensionContext context) throws Exception {
+        if (!initialized) {
+            if (!PropertyTestUtil.isLoggingSetupImminent()) {
+                System.err.println("!!! NonQuarkusTestLoggingEnabler");
+                IntegrationTestUtil.activateLogging();
```

Comment:
I would rather call `LoggingSetupRecorder` directly, but `handleFailedStart()` sounds weird.

Given that it's used in various non-failure cases, WDYT about renaming `handleFailedStart()` to `initializeBasicLogging()`? (or deprecate for removal and add the new method)

Reply:
Yeah, that makes sense

Reply:
Ok, do you think it's used outside of this repo? If yes we should play it safe and deprecate.

Reply:
I doubt it's used, but let's play it safe just in case